docker-compose.yml

```diff
@@ -147,7 +147,7 @@ services:
       ZOOKEEPER_SERVERS: zookeeper:12888:13888
 
   mongodb:
-    image: mongo:3.2
+    image: mongo:7.0
```

Comment:
We only test against mongo3 in this PR.

Reply:
I thought we agreed to add tests against both in the next PR?

Reply:
👍